_includes/live-demos/autocompleter/index.js

```diff
@@ -8,7 +8,6 @@ var specialChars = [
   { text: 'ampersand', value: '&' },
   { text: 'asterisk', value: '*' }
 ];
-
```

Comment:
This does decrease readability, but I guess if that's required by Jeykll then we'll have to live with it :cry: